backends/platform/sdl/posix/posix.h

```diff
@@ -28,7 +28,7 @@
 class OSystem_POSIX : public OSystem_SDL {
 public:
 	// Let the subclasses be able to change _baseConfigName in the constructor
-	OSystem_POSIX(Common::String baseConfigName = ".scummvmrc");
+	OSystem_POSIX(Common::String baseConfigName = "scummvm.ini");
```

Comment:
That looks to me a bit ugly. We provide here the config file name, but suddenly in OSystem constructor, we have .scummvmrc hardcoded.

From the other hand, I see now way how to address it cleanly.